src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ISymbolExtensions.cs

```diff
@@ -796,4 +796,12 @@ public static bool IsObsolete(this ISymbol symbol)
             MetadataName: nameof(ObsoleteAttribute),
             ContainingNamespace.Name: nameof(System),
         });
+
+    public static bool HasAttribute([NotNullWhen(true)] this ISymbol? symbol, INamedTypeSymbol? attributeClass)
```

Comment:
>NotNullWhen [](http://example.com/codeflow?start=37&length=11)

consider decorating the other parameter too

Reply:
done.